server/conn.go

```diff
@@ -155,8 +155,12 @@ type clientConn struct {
 
 func (cc *clientConn) String() string {
 	collationStr := mysql.Collations[cc.collation]
-	return fmt.Sprintf("id:%d, addr:%s status:%d, collation:%s, user:%s",
-		cc.connectionID, cc.bufReadConn.RemoteAddr(), cc.ctx.Status(), collationStr, cc.user,
+	autoCommit := 0
+	if cc.ctx.GetSessionVars().IsAutocommit() {
+		autoCommit = 1
+	}
+	return fmt.Sprintf("id:%d, addr:%s status:%d, collation:%s, user:%s autocommit:%d",
+		cc.connectionID, cc.bufReadConn.RemoteAddr(), cc.ctx.Status(), collationStr, cc.user, autoCommit,
```

Comment:
I think it is better to format the output of `cc.ctx.Status()`. Because it already contains the autocommit status.